chef_master/source/windows.rst

```diff
@@ -69,7 +69,7 @@ Spaces and Directories
 -----------------------------------------------------
 .. tag windows_spaces_and_directories
 
-Directories that are used by Chef on Windows cannot have spaces. For example, ``C:\Users\Steven Danno`` will not work, but ``C:\Users\StevenDanno`` will. Because of this, the ``knife supermarket install`` subcommand will fail if the directory contains a space.
+Directories that are used by Chef on Windows cannot have spaces. For example, ``C:\Users\User Name`` will not work, but ``C:\Users\UserName`` will. Because of this, the ``knife supermarket install`` subcommand will fail if the directory contains a space.
```

Comment:
I think we should talk generically about spaces and directories.  Instead of referring to `knife supermarket install` is a bit too specific... we should say something like 'You may encounter unexpected behavior if using Chef tools (such as knife) against a directory that contains a space.' or something along those lines.